CONTRIBUTING.md

```diff
@@ -32,14 +51,65 @@ real-time communication around OCIO. The group is invitation only, but just
 email us and we'll add anyone who is interested in participating in the
 discussion.
 
-* [GitHub Issues](https://github.com/imageworks/OpenColorIO/issues):
+* [GitHub Issues](https://github.com/AcademySoftwareFoundation/OpenColorIO/issues):
 GitHub **issues** are a great place to start a conversation! Issues aren’t
 restricted to bugs; we happily welcome feature requests and other suggestions
 submitted as issues. The only conversations we would direct away from issues are
 questions in the form of “How do I do X”. Please direct these to the ocio-dev or
 ocio-user mail lists, and consider contributing what you've learned to our
 docs if appropriate!
 
+## Legal Requirements
+
+OpenColorIO is a project of the Academy Software Foundation and follows the
+open source software best practice policies of the Linux Foundation.
+
+### License
+
+OpenColorIO is licensed under the [BSD-3-Clause](LICENSE.md)
+license. Contributions to the library should abide by that standard
+license.
+
+### Contributor License Agreements
+
+Developers who wish to contribute code to be considered for inclusion
+in OpenColorIO must first complete a **Contributor License Agreement**.
+
+OCIO uses EasyCLA for managing CLAs, which automatically checks to ensure CLAs
+are signed by a contributor before a commit can be merged.
+
+* If you are an individual writing the code on your own time and
+  you're SURE you are the sole owner of any intellectual property you
+  contribute, you can 
+  [sign the CLA as an individual contributor](https://github.com/communitybridge/easycla/blob/master/docs/Sign-a-CLA-as-an-Individual-Contributor-to-GitHub.md).
+
+* If you are writing the code as part of your job, or if there is any
+  possibility that your employers might think they own any
+  intellectual property you create, then you should use the 
+  [Corporate Contributor Licence Agreement](https://github.com/communitybridge/easycla/blob/master/docs/Contribute-to-a-GitHub-Company-Project.md).
+
+The OCIO CLA's are the standard forms used by Linux Foundation
+projects and
+[recommended by the ASWF TAC](https://github.com/AcademySoftwareFoundation/tac/blob/master/process/contributing.md#contributor-license-agreement-cla).
+
+### Commit Sign-Off
+
+Every commit must be signed off. That is, every commit log message must include
+a “`Signed-off-by`” line (generated, for example, with
```

Comment:
Presumably this needs to match up with something in the CLA, so it would be nice to have more detail here about exactly what should be in the sign-off string based on what settings were chosen in CLA Manager.

Reply:
I added an example and an explanation of how the sign-off ties into EasyCLA. @jmertic is my explanation correct?

Reply:
Yep, added a few minor tweaks but it looks like you got it right.

Reply:
Thanks!